pkg/distributor/ha_tracker.go

```diff
@@ -358,10 +358,10 @@ func (h *haTracker) cleanupOldReplicas(ctx context.Context, deadline time.Time)
 				continue
 			}
 
-			// We're blindly deleting a key here. It may happen that value was updated since we have read it few lines above,
-			// in which case Distributors will have updated value in memory, but Delete will remove it from KV store anyway.
+			// We're blindly deleting a key here. It may happen that value was updated since we have read it a few lines above,
+			// in which case distributors will have updated value in memory, but Delete will remove it from KV store anyway.
```

Comment:
```suggestion
			// We're blindly deleting a key here. It may happen that the value was updated since we have read it a few lines above,
			// in which case distributors will have updated value in memory, but Delete will remove it from the KV store anyway.
```